rails_multitenant.gemspec

```diff
@@ -14,8 +14,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/salsify/rails-multitenant'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.test_files    = Dir.glob('spec/**/*')
+  spec.files         = Dir['lib/**/*.rb', 'LICENSE.txt']
```

Comment:
As this gem isn't private and may be installed outside Salsify, maybe we should continue to include the "documentation" style files, namely README.md, CHANGELOG.md, CODE_OF_CONDUCT.md?

That said, I'm not sure anyone digs through their installed gems for those types of files these days, but rather looks on Github. So feel free to disregard.

Reply:
I don't think many people look at the content of gems now that most source is hosted in Github. I'm going to leave it out for now.